ports/lua/portfile.cmake

```
@@ -6,41 +6,52 @@ vcpkg_download_distfile(ARCHIVE
 vcpkg_extract_source_archive_ex(
     OUT_SOURCE_PATH SOURCE_PATH
     ARCHIVE ${ARCHIVE}
-    PATCHES vs2015-impl-c99.patch
+    PATCHES
+        vs2015-impl-c99.patch
+        fix-ios-system.patch
 )
 
 file(COPY ${CMAKE_CURRENT_LIST_DIR}/CMakeLists.txt DESTINATION ${SOURCE_PATH})
 
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+  FEATURES
+    # cpp COMPILE_AS_CPP
+  INVERTED_FEATURES
+    tools SKIP_INSTALL_TOOLS
+)
+if(VCPKG_TARGET_IS_IOS)
+    if("tools" IN_LIST FEATURES)
+        message(FATAL_ERROR "lua[tools] is not supported for iOS platform build")
+    endif()
+endif()
+
+set(ENABLE_LUA_CPP 0)
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
     OPTIONS
+        ${FEATURE_OPTIONS}
```

```suggestion
    OPTIONS_RELEASE
        ${FEATURE_OPTIONS}
    OPTIONS
```
And then re-include `-DSKIP_INSTALL_TOOLS=ON` so that we aren't building and installing tools in debug.